wazuh/wazuh-splunk
# Fix sorting on the Groups table

Summary
---------

Closes #1179 

This PR fixes the sorting of the Groups ctable columns. Non-sortable columns are no longer able to be sorted.

![image](https://user-images.githubusercontent.com/15186973/153656247-95908f35-46eb-4004-992c-2071b5decb39.png)


To test
---------
1. Go to the Management < Groups section
2. Check that only the 'name' column is sortable
